### ISSUE TITLE
Exclusive cases? -tta, -tti in MW

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/CORRECTIONS#24]
`^-` gives me only two maches in <a href="http://www.sanskrit-lexicon.uni-koeln.de/scans/MWScan/2014/web/webtc/servepdf.php?page=456">MW key2</a>.

Context
```
tOzAra
-tta
-tti
tma/n
```

I guess  `-tta, -tti` should be level 2, not level 1 - so no exclusive case at all.

**Comment by gasyoun** [comment]
@funderburkjim it's time to close it. But first a final decision has to be made. My Dec22 comment is still what I believe and what is too drastic remains so.

**Comment by funderburkjim** [comment]
I don't see an obvious thing to do.

Leave issue open until a definite solution is found, or until the problem is viewed as too minor to worry about.

**Comment by gasyoun** [comment]
If I have to manually clean (and I do) this item when compiling a new edition of my Reverse dictionary, I can't stay calm until this case is settled. Wonder what other users of MW would say, let me ask them per email.

**Comment by drdhaval2785** [comment]
@funderburkjim Long time since this issue started.
At that time we were super-conservative. 
Now we do print error corrections also with open mind.
Therefore, 2:1 decision goes in our favour.

If you want a neutral opinion still, @zaaf2 may be made referee.

**Comment by zaaf2** [comment]
त्त is not a suffix, but the past passive participle of दा (दात) with the elision of the root vowel and the consequent transformation of the voiced द् into the voiceless त्. The hyphen (-) just shows that this form is only found at the end of a compound. 
See Whitney Sanskrit Grammar 955 *f*. and 1087 *e*.:

![image](https://cloud.githubusercontent.com/assets/14730665/11218686/31383288-8d35-11e5-9c18-29d9c8c78b7e.png)

![image](https://cloud.githubusercontent.com/assets/14730665/11218691/35bef134-8d35-11e5-9bf2-1496ec283b1f.png)

As it is a word in its own right it is correct that it appears as a headword. As it begins with त्, it could not, according to the MW system, be placed under दा. 
In conclusion, I agree with @funderburkjim. There should be no change.

**Comment by gasyoun** [comment]
@zaaf2 your arguments are strong, still I hardly understand why only 2 such cases out of 160k.

**Comment by zaaf2** [comment]
As Whitney puts it, it is an "anomalously contracted form".  Exceptions are rare by nature.

**Comment by funderburkjim** [comment]
Thanks for the reference, @zaaf2 .  

I vote that this issue be closed.

**Comment by gasyoun** [comment]
Peter's answer was:
```
Well, it IS consistent as Jim pointed out, because the tta entry is in Devanagari.  Roman headwords are always subordinate to a preceding Devanagari headword.  If we make tta Roman, they will be improperly subordinated to the preceding Devanagari word.  On the other hand, now that this is a digital dictionary, alphabetic order has little relevance and we could just subordinate these words in our hierarchical list to dA1 'give', and tta also to dA3 'cut'.
I would let Jim and Thomas decide which way to go.
```
